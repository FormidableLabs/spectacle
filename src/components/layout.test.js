import React from 'react';
import { mount } from 'enzyme';
import { mountToJson } from 'enzyme-to-json';
import Layout from './layout';

describe('<Layout />', () => {
  test('should render correctly', () => {
    const wrapper = mount(<Layout><div>Content</div></Layout>);
    expect(mountToJson(wrapper)).toMatchSnapshot();
  });
});
