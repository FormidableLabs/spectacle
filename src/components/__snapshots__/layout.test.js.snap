// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> should render correctly 1`] = `
<Layout>
  <div
    data-radium={true}
    style={
      Object {
        "display": "flex",
      }
    }
  >
    <div>
      Content
    </div>
  </div>
</Layout>
`;
